mobile-app/app/screens/AppNavigator/screens/Portfolio/screens/ConvertScreenV2.tsx

```diff
@@ -0,0 +1,359 @@
+  const UTXOIcon = getNativeIcon('_UTXO')
+  return (
+    <ThemedTouchableOpacityV2
+      style={tailwind('border-0 pt-12 items-center flex-wrap')}
+      onPress={props.onPress} testID='button_convert_mode_toggle'
+    >
+      <UTXOIcon height={64} width={64} />
+      <ThemedViewV2
+        style={tailwind('absolute bottom-0 w-8 h-8 rounded-full items-center justify-center right-0 -mr-4')}
+        light={tailwind('bg-mono-light-v2-900')} dark={tailwind('bg-mono-dark-v2-900')}
+      >
+        <ThemedIcon
+          iconType='MaterialCommunityIcons' name='arrow-u-left-top' size={20}
+          light={tailwind('text-mono-dark-v2-900')} dark={tailwind('text-mono-light-v2-900')}
+        />
+      </ThemedViewV2>
+    </ThemedTouchableOpacityV2>
+  )
+}
+
+function ConversionLabel (props: { sourceUnit: string, targetUnit: string }): JSX.Element {
+  return (
+    <View style={tailwind('mt-1 flex-row items-center')}>
+      <ThemedTextV2
+        style={tailwind('font-normal-v2 text-xs')} testID='convert_source'
+        light={tailwind('text-mono-light-v2-700')} dark={tailwind('text-mono-dark-v2-700')}
+      >
+        {`(${translate('screens/ConvertScreen', props.sourceUnit)}`}
+      </ThemedTextV2>
+      <ThemedIcon
+        iconType='Feather' name='arrow-right' style={tailwind('px-1')}
+        light={tailwind('text-mono-light-v2-700')} dark={tailwind('text-mono-dark-v2-700')}
+      />
+      <ThemedTextV2
+        style={tailwind('font-normal-v2 text-xs')} testID='convert_target'
+        light={tailwind('text-mono-light-v2-700')} dark={tailwind('text-mono-dark-v2-700')}
+      >
+        {`${translate('screens/ConvertScreen', props.targetUnit)})`}
+      </ThemedTextV2>
+    </View>
+  )
+}
+
+function ConversionResultCard (props: { unit: string | undefined, convertAmount: string, targetAmount: string }): JSX.Element {
+  return (
+    <ThemedViewV2
+      style={tailwind('flex-col w-full p-5 mt-6 rounded-lg-v2 border-0.5')} testID='convert_result_card'
+      light={tailwind('border-mono-light-v2-300')} dark={tailwind('border-mono-dark-v2-300')}
+    >
+      <ThemedViewV2
+        style={tailwind('flex-row border-b-0.5 items-center pb-5')}
+        light={tailwind('border-mono-light-v2-300')} dark={tailwind('border-mono-dark-v2-300')}
+      >
+        <ThemedTextV2
+          style={tailwind('font-normal-v2 text-sm pr-2')} testID='convert_available_label'
+          light={tailwind('text-mono-light-v2-500')} dark={tailwind('text-mono-dark-v2-500')}
+        >
+          {`${translate('screens/ConvertScreen', 'Available {{unit}}', { unit: props.unit })}`}
+        </ThemedTextV2>
+        <ThemedTextV2
+          style={tailwind('flex-1 font-normal-v2 text-sm text-right')} testID='convert_available_amount'
+          light={tailwind('text-mono-light-v2-800')} dark={tailwind('text-mono-dark-v2-800')}
+        >
+          {new BigNumber(props.convertAmount).toFixed(8)}
```

Comment:
Available UTXO is displaying the token balance
![image](https://user-images.githubusercontent.com/13292662/183318955-b90a5f6c-c94b-4c6e-80ef-ef69f31d6906.png)

Reply:
Done, fixed Available UTXO/tokens display amount